encodings/alp/src/alp/mod.rs

```diff
@@ -269,12 +268,23 @@ fn encode_chunk_unchecked<T: ALPFloat>(
     }
```

Comment:
I'm not convinced this fill logic adds values. We are picking a value but we could do it statically and I doubt anything would change

Reply:
made pr #2560 to see if there's any practical difference